tools/topology/topology2/platform/intel/dmic-default.conf

```diff
@@ -40,4 +40,12 @@ Define {
 	DMIC0_RATE                      48000
 	DMIC1_RATE                      16000
 	DMIC1_ENABLE			"false"
+
+	# TDFB pipeline
+	DMIC0_ENHANCED_CAPTURE		"false"
+	DMIC0_TDFB_PCM_ID		200
```

Comment:
Could I change this to e.g. 50, or something else after 49?

Reply:
That sounds good. But how to handle other beamformer enhanced endpoints, HDA analog and SDW? I would assume there's only single such capture endpoint in a device, but having dedicated PCMn might be more safe?

Reply:
I am not sure we want to deal with HDA analog capture, this could be super confusing.

For SoundWire it's a bit complicated. We only support 2ch capture, and we don't have any indication that there are 4 mics. It's been a problem for a while. And then we have lots of other issues to sort out, I am not super hot on adding new capabilities just now while we're having to deal with all kinds of ACPI/machine/codec variations.

Reply:
I'm not targeting to add features to SDW platfoms for v2.10, the code freeze is soon and I don't have time to test nor any devices to see myself that it works.